packages/rn-tester/js/examples/FlatList/FlatListExample.js

```diff
@@ -74,6 +78,10 @@ class FlatListExample extends React.PureComponent<Props, State> {
     empty: false,
     useFlatListItemComponent: false,
     fadingEdgeLength: 0,
+    header: false,
+    footer: false,
```

Comment:
@stealthanthrax I think it would be better to set them as `true` initially? If the users don't interact with the header and footer toggles, they won't be able to see the header/footer in action at all.